--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts
@@ -60,9 +60,24 @@
 &rp1_usb0 {
 	pinctrl-0 = <&usb_vbus_default_state>;
 	pinctrl-names = "default";
 	status = "okay";
 };
 
 &rp1_usb1 {
 	status = "okay";
 };
+
+&rp1_spi0 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&rp1_spi0_gpio9 &rp1_spi0_cs_gpio7>;
+	cs-gpios = <&rp1_gpio 8 GPIO_ACTIVE_LOW>,
+		   <&rp1_gpio 7 GPIO_ACTIVE_LOW>;
+	status = "okay";
+
+	tpm@1 {
+		compatible = "infineon,slb9670", "tcg,tpm_tis-spi";
+		reg = <1>; /* CE1 */
+		spi-max-frequency = <32000000>;
+		status = "okay";
+	};
+};
